cfg/file-name-conventions.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# Options: snake_case, kebab-case, camelCase, PascalCase, UPPER_CASE,
+# and * for any case.
+default: 'kebab-case'
+filetypes:
+  js: 'kebab-case'
+  md: '*'
+  move: 'snake_case'
+  py: 'kebab-case'
+  rs: 'snake_case'
+  toml: '*'
```

Comment:
@alnoki so I did end up just setting toml to `*` because` pyproject.toml` isn't PascalCase. I'm not sure if we wanted to make this more robust (allow for multiple cases for one file type?) or just keep it simple and use it like this.

Reply:
How about still using `PascalCase` for general case for `toml`, but allowing escapes for specific violations, e.g. for `src/python/hooks/pyproject.toml`?

Reply:
Done! Added an `ignore_files` list for a list of file names to ignore